diff --git a/gui/qt/__init__.py b/gui/qt/__init__.py
index 7a0fcb6..2efb836 100644
--- a/gui/qt/__init__.py
+++ b/gui/qt/__init__.py
@@ -86,6 +86,8 @@ class QNetworkUpdatedSignalObject(QObject):
 class ElectrumGui:
 
     def __init__(self, config, daemon, plugins):
+        if config.get('auto_connect') is None:
+            config.set_key("language", "ja_JP", True)
         set_language(config.get('language'))
         # Uncomment this call to verify objects are being properly
         # GC-ed when windows are closed
